lldb/test/API/repl/clang/TestClangREPL.py

```diff
@@ -54,3 +55,16 @@ def test_basic_completion(self):
         self.expect_repl("$persistent + 10", substrs=["(long) $2 = 17"])
 
         self.quit()
+
+    # PExpect uses many timeouts internally and doesn't play well
+    # under ASAN on a loaded machine..
+    @skipIfAsan
+    @skipIf(oslist=["linux"], archs=["arm", "aarch64"])  # Randomly fails on buildbot
```

Comment:
Is there some paper trail here, was this related to the expression or to buildbot setup? If it's not for sure because of the expression then leave the skip as is, It's probably a machine load problem like the comment for ASAN says.

Reply:
I actually don't know. The existing test already has issues with basic expr eval, and the new test will have them as well, so I decided to just copy the same configuration.

Reply:
Fair enough, if there's no specific reason I bet it's a relic from when we had way too many bots on the same machine. Today we don't have as many as we used to, but not your problem to figure out. It's being run somewhere that's the main thing.